server/portal/libs/agave/operations.py

```diff
@@ -452,15 +452,9 @@ def upload(client, system, path, uploaded_file):
     token = client.access_token.access_token
     systemId = system
     dest_path = os.path.join(path.strip('/'), uploaded_file.name)
-
-    res = r.post(
-        url=f'{base_url}/v3/files/ops/{systemId}/{dest_path}',
-        files={"file": uploaded_file.file},
-        headers={"X-Tapis-Token": token})
-
-    res.raise_for_status()
-
-    response_json = res.json()
+    
+    response_json = client.files.insert(systemId=systemId, path = dest_path, file = uploaded_file)
```

Comment:
requires code formatting. And unit tests are failing, please see the PR checks.

Reply:
@asimregmi Have a look here for linting tips https://github.com/TACC/Core-Portal#linting-and-formatting-conventions

Reply:
fixed the linting and formatting and updated unit tests to match new code logic. Also updated the `server/conftest.py` file to update `agave_file_mock` to `tapis_file_mock`